src/main/java/org/zalando/nakadi/service/EventStreamWriterBinary.java

```diff
@@ -85,6 +94,31 @@ public int writeBatch(final OutputStream os, final Cursor cursor, final List<byt
         return byteCount;
     }
 
+    private int writeDebugInfo(final OutputStream os) throws IOException {
+        int byteCount = 0;
+
+        os.write(B_DEBUG_BEGIN);
+        byteCount += B_DEBUG_BEGIN.length;
+
+        final DateTime now = new DateTime(DateTimeZone.UTC);
+        final InetAddress ip;
+        try {
+            ip = InetAddress.getLocalHost();
+        } catch (final UnknownHostException e) {
+            throw new MyNakadiRuntimeException1("problem getting servers ip", e);
+        }
+        final String debugString = "Batch written at " + now.toString() + " from server " + ip.getHostAddress();
```

Comment:
Maybe it will make sense to write it as a structured json, like
```
{ 
  "sent_at": "2017-08-15T13:57:07.634Z",
  "nakadi_ip": "145.34.12.89"
}
```
That will allow to parse it easier in consumer (but I actually I don't know if we will want to parse it)